apps/backoffice/app/cadence.css

```
@@ -0,0 +1,219 @@
+
+  /* Error / alert */
+  --cad-rose: oklch(0.6 0.2 22);
+  --cad-rose-bg: oklch(0.945 0.04 20);
+
+  /* A single, very subtle elevation — used sparingly */
+  --cad-shadow: 0 1px 2px oklch(0.3 0.02 60 / 5%);
+
+  color-scheme: light;
+  font-family: var(--cad-font-sans);
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.13.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)